.pre-commit-config.yaml

```diff
@@ -26,13 +26,16 @@ repos:
             'flake8-no-pep420==2.0.0',
         ]
         args: ["--config=.flake8"]
+        exclude: '/(migrations|\.direnv|\.eggs|\.git|\.hg|\.mypy_cache|\.nox|\.tox|\.venv|venv|\.env|env|\.svn|_build|buck-out|build|dist)/'
   - repo: https://github.com/PyCQA/isort
     rev: 5.10.1
     hooks:
       - id: isort
         args: ["--settings-file=backend/pyproject.toml"]
+        exclude: '/(migrations|\.direnv|\.eggs|\.git|\.hg|\.mypy_cache|\.nox|\.tox|\.venv|venv|\.env|env|\.svn|_build|buck-out|build|dist)/'
   - repo: https://github.com/ambv/black
     rev: 22.3.0
     hooks:
       - id: black
         args: ["--config=backend/pyproject.toml"]
+        exclude: '/(migrations|\.direnv|\.eggs|\.git|\.hg|\.mypy_cache|\.nox|\.tox|\.venv|venv|\.env|env|\.svn|_build|buck-out|build|dist)/'
```

Comment:
Did you try changing `exclude` option in  `[tool.black]` section of `pyproject.toml` to `force-exclude` as I mentioned in a comment earlier?

Reply:
If it works out with `force-exclude`, this regex will also turn out to be redundant.

Reply:
Yes, it seems it works with `force-exclude`. I've updated the PR.